### ISSUE TITLE
Disabled radio buttons

### ISSUE THREAD

**Issue by chris-gds** [Financial-Times/origami#1773]
## What?

Consider a redesign of component

## Why?

Consider failing contrast [WCAG 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum) and blog post: "[disabled buttons suck](https://axesslab.com/disabled-buttons-suck/)"

## Screenshots

<img width="221" alt="FT design system origami disabled radio buttons highlighting colour contrast issues within disabled radios" src="https://github.com/user-attachments/assets/12e305bd-5be3-49d6-851d-cb94c312cf2d">
